projects/js-packages/components/components/icon-tooltip/stories/index.tsx

```diff
@@ -0,0 +1,60 @@
+import IconTooltip from '../index';
+import type { Placement } from '../types';
+
+const AVAILABLE_PLACEMENTS: Placement[] = [
+	'top',
+	'top-start',
+	'top-end',
+	'bottom',
+	'bottom-start',
+	'bottom-end',
+];
+
+export default {
+	title: 'JS Packages/Components/IconTooltip',
+	component: IconTooltip,
+	argTypes: {
+		className: {
+			control: { type: 'text' },
+		},
+		iconClassName: {
+			control: { type: 'text' },
+		},
+		placement: {
+			control: { type: 'select' },
+			options: AVAILABLE_PLACEMENTS,
+		},
+		animate: {
+			control: { type: 'boolean' },
+		},
+		iconCode: {
+			control: { type: 'text' },
+		},
+		title: {
+			control: { type: 'text' },
+		},
+		children: {
+			control: { type: null },
+		},
+	},
+};
+
+const Template = args => (
+	<div style={ { position: 'absolute', height: '2000px', left: '400px', top: '400px' } }>
+		<IconTooltip { ...args } />
+	</div>
+);
```

Comment:
The styling on this DIV doesn't play nice with Storybook. For example, the controls are offscreen in the Docs view or on a smaller screen. Could we maybe position this closer to the top-left of the canvas?

Reply:
Yes, @a8ck3n! The original intention of leaving these styles is to ensure the `position` in the Storybook will work as the player set. I would move it to the more appropriate edge.